docs/overview/index.md

```diff
@@ -3,19 +3,17 @@ id: overview_index
 title: "Summary"
 ---
 
-ZIO Prelude is an alternative approach to functional abstractions in Scala, which throws out the classic functor hierarchy in favor of a modular algebraic approach that is smaller, easier to understand and teach, and more expressive.
+ZIO Prelude is a library focused on providing a core set of functional data types and abstractions that can help you solve a variety of day to day problems. The tools provided by ZIO Prelude fall into the following main categories:
```

Comment:
this is very compelling: focus on what it is and nothing else, don't distract us about what it is not.